m0t1fs/linux_kernel/st/m0t1fs_common_inc.sh

```diff
@@ -622,20 +632,22 @@ function build_conf()
  # pools, racks, enclosures, controllers and their versioned objects.
 	echo -e "
 [$(($IOS_OBJS_NR + $((${#mdservices[*]} * 5)) + $NR_IOS_DEVS + 19
-    + $MD_OBJ_COUNT + $PVER1_OBJ_COUNT + 5 + $DIX_PVER_OBJ_COUNT)):
+    + $MD_OBJ_COUNT + $PVER1_OBJ_COUNT + 5 + $DIX_PVER_OBJ_COUNT + $FDMI_ITEMS_NR)):
```

Comment:
It is used for all other variables. To keep it the same as others.

This Codacy warning is false.